docs/src/app/components/pages/components/date-picker.jsx

```diff
@@ -140,6 +140,22 @@ class DatePickerPage extends React.Component {
             'the change the first argument will always be null and the second ' +
             'argument will be the new Date instance.'
           },
+          {
+            name: 'onFocus',
+            header: 'function(e)',
```

Comment:
Yes, sure - that format was copied from elsewhere in the docs. I'll find the culprit and fix it there too.